src/examples/colors/complexity.py

```diff
@@ -58,53 +75,111 @@ def generate_color_complexity(nat_langs, art_langs, meaning_dist, prior, use_rkk
 
     current_dir = os.path.dirname(os.path.realpath(__file__))
     import matplotlib.pyplot as plt
-    #Generate the imshow heatmap for the meaning
-    plt.imshow(meaning_dist,  cmap="hot")
-    plt.savefig(f"{current_dir}/outputs/meaning_dists.jpg")
 
+    # Generate the imshow heatmap for the meaning
+    plt.imshow(meaning_dist, cmap="hot")
+    plt.savefig(f"{current_dir}/outputs/meaning_dists.jpg")
 
     current_dir = os.path.dirname(os.path.realpath(__file__))
-    if(nat_langs is None or art_langs is None):
+    if nat_langs is None or art_langs is None:
         print("Natural languages not provided, reading from file")
-        nat_langs:dict[str, Language] = pickle.load(open(f"{current_dir}/outputs/natural-languages.pkl", "rb"))
-        art_langs:dict[str, Language] = pickle.load(open(f"{current_dir}/outputs/artificial-languages.pkl", "rb"))
-
-    #Generate the meaning/accuracy/complexity for all languages based on the prior, meaning and Language
+        nat_langs: dict[str, Language] = pickle.load(
+            open(f"{current_dir}/outputs/natural-languages.pkl", "rb")
+        )
+        art_langs: dict[str, Language] = pickle.load(
+            open(f"{current_dir}/outputs/artificial-languages.pkl", "rb")
+        )
+
+    # Generate the meaning/accuracy/complexity for all languages based on the prior, meaning and Language
     language_data = []
     for language in nat_langs:
-        #Dereference the lang code to get the actual language associated with it
+        # Dereference the lang code to get the actual language associated with it
         language_name = language.name
         lang_code = int(language.lang_code)
-        #Exclude the languages Amuzgo, Camsa, Candoshi, Chayahuita, Chiquitano, Cree, Garífuna (Black Carib), Ifugao, Micmac, Nahuatl, Papago (O’odham), Slave, Tacana, Tarahumara (Central), Tarahumara (Western).
-        #These languages have fewer than 5 definitions for major terms
-        excluded_language_codes = [7, 19, 20, 25, 27, 31, 38, 48, 70, 78, 80, 88, 91, 92, 93]
-        uniform_prior = np.array([1/len(meaning_dist)]*len(meaning_dist))
+        # Exclude the languages Amuzgo, Camsa, Candoshi, Chayahuita, Chiquitano, Cree, Garífuna (Black Carib), Ifugao, Micmac, Nahuatl, Papago (O’odham), Slave, Tacana, Tarahumara (Central), Tarahumara (Western).
+        # These languages have fewer than 5 definitions for major terms
+        excluded_language_codes = [
```

Comment:
Can you have a script that identifies these langs and writes the IDs to a file, and then read that file in to get this list?  It feels ad hoc to have the IDs hard-coded in here